src/frontend/src/pages/SearchPage.jsx

```diff
@@ -43,85 +42,66 @@ export default function SearchPage() {
 
   const { showToastMessage } = useToast();
 
-  const applyFilters = useCallback((resultsToFilter, criteria) => {
-    const { type, year } = criteria;
-    const filtered = resultsToFilter.filter((item) => {
-      const matchesType = type ? item.type === type : true;
-      const matchesYear = year ? item.releaseYear === year : true;
-      return matchesType && matchesYear;
-    });
-    setFilteredResults(filtered);
-  }, []);
-
-  const handleFilterChange = useCallback(
-    (newFilterCriteria) => {
-      setFilterCriteria(newFilterCriteria);
-      applyFilters(mediaResults, newFilterCriteria);
-    },
-    [mediaResults, applyFilters],
-  );
-
   const handleSearch = (query_data) => {
     setMediaPage({ ...mediaPage, page: 1 });
+    setPersonPage({ ...personPage, page: 1 });
     setQuery(query_data);
   };
 
-  const performSearch = useCallback(
-    async (query_data) => {
-      setLoading(true);
+  const performMediaSearch = useCallback(
+    async (query_data = true) => {
       let params = {
         ...query_data,
-        ...filterCriteria,
         page: mediaPage.page,
         count: mediaPage.count,
       };
+
       try {
         const response = await fetchMedia(params);
         setMediaResults(response);
-        applyFilters(response.items, filterCriteria);
+      } catch (error) {
+        showToastMessage(
+          error.message || 'Error occurred while searching media.',
+          'danger',
+        );
+        setMediaResults({ items: [], numberOfItems: 0 });
+      }
+    },
+    [mediaPage.count, mediaPage.page, showToastMessage],
+  );
+
+  const performPersonSearch = useCallback(
+    async (query_data) => {
+      let params = {
+        name: query_data.query,
+        page: personPage.page,
+        count: personPage.count,
+      };
 
-        if (params.query_type !== 'Structured') {
-          let q = params.query;
-          if (params.query_type !== 'Simple') {
-            q = params.keywords.join(' ');
-          }
```

Comment:
Person query should only happen if not structured search, maybe add an `if` statement to the effect to ensure its not called if structured search, and is cleared instead